warehouse/templates/legacy/api/simple/detail.html

```diff
@@ -19,7 +19,7 @@
   <body>
     <h1>Links for {{ project.name }}</h1>
     {% for file in files -%}
-    <a href="{{ request.route_url('packaging.file', path=file.path) }}#sha256={{ file.sha256_digest }}"{% if file.release.requires_python %} data-requires-python="{{ file.release.requires_python }}"{% endif %}>{{ file.filename }}</a><br/>
+    <a href="{{ request.route_url('packaging.file', path=file.path) }}#sha256={{ file.sha256_digest }}"{% if file.release.requires_python %} data-requires-python="{{ file.release.requires_python }}"{% endif %}{% if file.release.yanked %}data-yanked{%if file.release.yanked %}="file.release.yanked"{% endif %}{% endif %}>{{ file.filename }}</a><br/>
```

Comment:
This conditional looks weird, why is there two identical conditionals nested inside each other? Why are we setting the ``data-yanked`` attribute equal to the string ``file.release.yanked``?

Reply:
Looks like that was a holdover from https://github.com/pypa/warehouse/pull/5838/commits/f976d0abf1156990a5c34db8b8130fcde1796f02, where we might have had a comment or something? I'm not sure what your original intention was, but I updated this just to show the attribute when the release is yanked.